### ISSUE TITLE
Bitfinex2 IOTA/BTC pair is not unified properly

### ISSUE THREAD

**Issue by nmiculinic** [ccxt/ccxt#830]
```
                'IOT/BTC': {'id': 'tIOTBTC', 'symbol': 'IOT/BTC', 'base': 'IOT', 'quote': 'BTC'},
```

similarly to other IOTA pairs on Bitfinex; they all have unified wrong symbol name, which should be IOTA/BTC

**Comment by nmiculinic** [comment]
Also withdrawal fee is out of date                        'BTC': 0.0005; right now it's 0.0015 BTC

**Comment by kroitor** [comment]
> similarly to other IOTA pairs on Bitfinex; they all have unified wrong symbol name, which should be IOTA/BTC

This was fixed here: https://github.com/ccxt/ccxt/issues/796 (3 days before)
Please update to most recent version of ccxt. The issue should go away.

> Also withdrawal fee is out of date 'BTC': 0.0005; right now it's 0.0015 BTC

Those were added 20 minutes ago in this PR: https://github.com/ccxt/ccxt/pull/811, we will update them. Thx!

**Comment by illera88** [comment]
Fees are gotten from https://www.bitfinex.com/fees

**Comment by kroitor** [comment]
@illera88 are you sure that page is up to date?

**Comment by illera88** [comment]
That's the official website and fee site from Bitfinex. I haven't make an actual withdraw to test it tho.

**Comment by kroitor** [comment]
@nmiculinic sorry, my bad, the bitfinex2 pairs are indeed not consistent with IOT/IOTA, will fix right away.

**Comment by kroitor** [comment]
@illera88 that page looks like a list of outdated tx fees to me... if it does not correspond to truth, we will have to revert it back to undefined and scrape it from their website somehow... got any ideas?

**Comment by illera88** [comment]
The fee for BTC from their own API says that the BTC fee is '0.0005' too. 

you can get it using `private_post_account_fees()`

![2017-12-12 00_49_59-bender debugging - microsoft visual studio](https://user-images.githubusercontent.com/5193128/33839770-76c85568-ded6-11e7-9bfe-37b7de3f9198.png)

**Comment by kroitor** [comment]
We will strip down those hardcoded values for the private method then. It will be unified as `fetchFees()` and will populate the fee info...

**Comment by illera88** [comment]
the hardcoded values are correct. I agree is better using the exchange API to get the values if that is possible but I don't see how leaving the hardcoded values (which are correct) is gonna harm until 'fetchFees()' is implemented

**Comment by kroitor** [comment]
@illera88 if they are up to date, we can, of course, keep them, until we have fetchFees. The only reason for not having them is not to confuse users, if Bitfinex decides to change them once again after BTC skyrockets to $100k. In fact, it's the txFee itself that does change over time...

**Comment by kroitor** [comment]
@nmiculinic the IOTA mapping is fixed in 1.10.357. Thx for reporting!
